plugins/python_3_script/plugin.spec.yaml

```diff
@@ -6,7 +6,7 @@ title: Python 3 Script
 vendor: rapid7
 support: rapid7
 status: []
-description: Python is a programming language that lets you work quickly and integrate systems more effectively. This plugin allows you to run Python 3 code. It includes Python 3.8.1 and its standard library as well as the following 3rd party libraries
+description: Python is a programming language that lets you work quickly and integrate systems more effectively. This plugin allows you to run Python 3 code. It includes Python 3.9.18 and its standard library as well as the following 3rd party libraries
```

Comment:
is it worthwhile if you're making more changes to Py3 to specify the full SDK version? It means going forward (until we come up with a better way) we will easily see this is using the specific SDK version?

Reply:
Sounds good. I think this may be added to the `plugin.spec.yaml` file 😄